src/editor/Sidebar.js

```diff
@@ -156,13 +161,35 @@ const Sidebar = ( {
 				</>
 			) }
 			{ placement === 'inline' && (
-				<RangeControl
-					label={ __( 'Approximate Position (in percent)', 'newspack-popups' ) }
-					value={ trigger_scroll_progress }
-					onChange={ value => onMetaFieldChange( 'trigger_scroll_progress', value ) }
-					min={ 0 }
-					max={ 100 }
-				/>
+				<>
+					<SelectControl
+						label={ __( 'Trigger', 'newspack-popups' ) }
+						help={ __( 'The event to trigger the prompt.', 'newspack-popups' ) }
+						value={ trigger_type }
+						options={ [
+							{ label: __( 'Scroll Progress', 'newspack-popups' ), value: 'scroll' },
```

Comment:
"Scroll progress" is more suitable in the case of overlay prompts, where the action of scrolling is triggering the prompt to be displayed. Here it's just about the insertion position.

```suggestion
							{ label: __( 'Percentage', 'newspack-popups' ), value: 'scroll' },
```